src/dnx.clr/KatanaManager.h

```diff
@@ -105,13 +76,9 @@ class KatanaManager :
 
         _applicationBase = applicationBase;
 
-        _HR(CLRCreateInstance(CLSID_CLRMetaHost, PPV(&_MetaHost)));
         _HR(CLRCreateInstance(CLSID_CLRMetaHostPolicy, PPV(&_MetaHostPolicy)));
 
-        IStreamPtr cfgStream = new ComObject<FileStream>();
-        _HR(static_cast<FileStream*>(cfgStream.GetInterfacePtr())->Open(_clrConfigFilePath));
-
-        WCHAR wzVersion[130] = { 0 };
+        WCHAR wzVersion[130] = L"v4.0.30319";
```

Comment:
```
wzVersion[0] = L"v";
wzVersion[1] = L"4";
wzVersion[2] = L".";
wzVersion[3] = L"0";
wzVersion[4] = L".";
...
```
Before it was just `wzVersion[0] = 0;` indicating the empty string since the version was passed in the config file.